clang/test/SemaCXX/verbose-trap.cpp

```diff
@@ -0,0 +1,28 @@
+// RUN: %clang_cc1 -std=c++11 -fsyntax-only -fcxx-exceptions -verify %s
+
+#if !__has_builtin(__builtin_verbose_trap)
+#error
+#endif
```

Comment:
Is this necessary?

Reply:
This just makes sure the builtin is supported unconditionally and doesn't depend on some features or anything (cc @ldionne ).